Nos/Views/ActionButton.swift

```diff
@@ -138,8 +138,8 @@ struct ActionButtonStyle: ButtonStyle {
                     ZStack {
                         LinearGradient(
                             colors: [
-                                Color(red: 1, green: 1, blue: 1, opacity: 0.2),
-                                Color(red: 1, green: 1, blue: 1, opacity: 1.0),
+                                Color.actionButtonStyleGradientLeading,
+                                Color.actionButtonStyleGradientTrailing
```

Comment:
I'd suggest a rename for these to be more accurate. I know the assets will need to be renamed, but it's easier for me to see here. I'd go with something like this:

```suggestion
                                Color.actionButtonBackgroundGradientTop,
                                Color.actionButtonBackgroundGradientBottom
```